cpp/vm/evmzero/interpreter_test.cc

```diff
@@ -2978,15 +2978,15 @@ TEST(InterpreterTest, DIFFICULTY) {
   });
 }
 
-TEST(InterpreterTest, DIFFICULTY_OutOfGas) {
+TEST(InterpreterTest, PREVRANDAO_OutOfGas) {
   RunInterpreterTest({
-      .code = {op::DIFFICULTY},
+      .code = {op::PREVRANDAO},
       .state_after = RunState::kErrorGas,
       .gas_before = 1,
   });
 }
 
-TEST(InterpreterTest, DISABLED_DIFFICULTY_StackOverflow) {}
+TEST(InterpreterTest, DISABLED_PREVRANDAO_StackOverflow) {}
```

Comment:
This test is useless -- either we fix it or remove it.

Reply:
There are 14 other disabled stack overflow tests, I think they should be handled in a separate PR.